pywinauto/base_wrapper.py

```diff
@@ -684,6 +680,10 @@ def click_input(
                           ','.join([str(coord) for coord in coords]) + ')'
             ActionLogger().log(message)
```

Comment:
I'd suggest to call `ActionLogger().log(message)` after actual click, though message must be formed before the action.

The rest looks fine.

Reply:
Because all other methods log something after action, so it looks more consistent.